src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Reader {

```

Comment:
better way to separate logic is to create several methods in `WorkWithFile`, because right now your `Reader` not only reads data, but also calculates statistics